crates/spark/src/services/lightning.rs

```diff
@@ -225,6 +225,40 @@ impl<S: Signer> LightningService<S> {
         preimage: Option<Vec<u8>>,
         expiry_secs: Option<u32>,
     ) -> Result<LightningReceivePayment, ServiceError> {
+        self.create_lightning_invoice_internal(amount_sats, memo, preimage, expiry_secs, None)
+            .await
+    }
+
+    pub async fn create_lightning_invoice_for_pubkey(
```

Comment:
Thanks @benthecarman just a small suggestion. Perhaps instead of adding another function we should add an optional publicKey parameter to the existing create_lightning_invoice? If passed None we will use the pubkey from the signer. 
WDYT?